fitnessllm_dataplatform/utils/task_utils.py

```
@@ -0,0 +1,52 @@
+        return data.isoformat()
+    return data
+
+
+def get_schema_path(data_source: FitnessLLMDataSource | None, data_stream: FitnessLLMDataStream | None) -> str:
+    if data_source and data_stream:
+        return f"fitnessllm_dataplatform/stream/{data_source.value.lower()}/schemas/{data_stream.value.lower()}.json"
+    return "fitnessllm_dataplatform/schemas/metrics.json"
+
+
+def load_schema_from_json(data_source: FitnessLLMDataSource, data_stream: FitnessLLMDataStream) -> list[bigquery.SchemaField]:
+    with open(get_schema_path(data_source, data_stream), 'r') as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)